Binary files wxorig/.DS_Store and wxnew/.DS_Store differ
Binary files wxorig/src/.DS_Store and wxnew/src/.DS_Store differ
diff -uNr wxorig/src/common/docview.cpp wxnew/src/common/docview.cpp
--- wxorig/src/common/docview.cpp	2009-10-09 23:30:10.000000000 -0400
+++ wxnew/src/common/docview.cpp	2009-11-05 11:00:02.000000000 -0500
@@ -1372,6 +1372,15 @@
     if ( !(flags & wxDOC_NEW) && temp->FileMatchesTemplate(path) )
         AddFileToHistory(path);
 
+    // Activate the document to bring it to the forefront
+    if ( docNew->GetFirstView() )
+    {
+        docNew->GetFirstView()->Activate(true);
+        ActivateView(docNew->GetFirstView());
+        if ( docNew->GetDocumentWindow() )
+            docNew->GetDocumentWindow()->SetFocus();
+    }
+
     return docNew;
 }
 
